augur/io/sequences.py

```diff
@@ -2,10 +2,28 @@
 import os
 
 from augur.errors import AugurError
+from importlib.metadata import version
```

Comment:
_non-blocking_

Having different things that differ only in case is a potent source of potential confusion, maybe this instead?

```suggestion
from importlib.metadata import version as metadata_version
```

Reply:
Aliased as `installed_version` which improves readability of the usage:

https://github.com/nextstrain/augur/blob/db6fe9f1cce13b8bc7114b131580c3c9209c00f7/augur/io/sequences.py#L20-L22

Reply:
yeah, that's good i like that.